crates/driver/src/infra/config/file/mod.rs

```diff
@@ -574,3 +577,61 @@ pub enum GasEstimatorType {
     Native,
     Web3,
 }
+
+#[derive(Debug, Deserialize)]
+#[serde(rename_all = "kebab-case", deny_unknown_fields)]
+pub struct OrderPriorityConfig {
+    /// Defines order prioritization strategies that will be applied in the
+    /// specified order.
+    #[serde(default = "default_order_priority_strategies")]
+    pub strategies: Vec<OrderPriorityStrategy>,
+
+    /// Configures the threshold for `CreationTimestamp` order prioritization
+    /// strategy. Orders created within this threshold will be prioritized.
+    /// Takes effect only if `CreationTimestamp` is specified in
+    /// `order_priority_strategies`.
+    #[serde(default = "default_order_creation_timestamp_threshold")]
+    pub order_creation_timestamp_threshold: Duration,
```

Comment:
Seems like this should be only part of the respective strategy.

Reply:
I actually tried to make it part of the strategy, but that produced a boilerplate since `serde` is not as suitable for this as `clap`, for example. I will try to recover it to decide whether it is worth making this change.

Reply:
Refactored the config https://github.com/cowprotocol/services/pull/2872/commits/badcbb62261b6c10f04066078e41b8d6cac37b4b